src/stdlib_stats_distribution_PRNG.fypp

```diff
@@ -0,0 +1,212 @@
+#:include "common.fypp"
+module stdlib_stats_distribution_PRNG
+    use stdlib_kinds, only: int8, int16, int32, int64
+    implicit none
+    private
+    integer, parameter :: MAX_INT_BIT_SIZE = bit_size(1_int64)
+    integer(int64), save :: st(4), si = 614872703977525537_int64
+    logical, save :: seed_initialized = .false.
+
+    public :: random_seed
+    public :: dist_rand
+    public :: jump
+    public :: long_jump
```

Comment:
These two functions are not mentioned in the specs. I understand how they could be used. However, an explanation and example in the specs are needed. They must also appear in the tests.

Reply:
As said in above, the jump and long_jump functions are not mean for users' applications yet. Included here for completeness.

Reply:
Removed two procedures in this PR.